pkg/console/subresource/configmap/configmap.go

```diff
@@ -155,14 +159,20 @@ func servingInfo() yaml.MapSlice {
 	}
 }
 
-func customization(brand operatorv1.Brand, docURL string) map[string]string {
+func customization(brand operatorv1.Brand, docURL string, customProductName string, customLogoFile string) map[string]string {
```

Comment:
nit: same inversion as above, this `func` is lower down the stack, doesn't know if its custom or not.  Just `productName`, and `logoFile`.